submitted assignments/Vishakha_Deshmukh/script.js

```
@@ -0,0 +1,451 @@
+class InvoiceComponent extends HTMLElement {
+    // Default configuration with class names for various elements
+    defaultConfig = {
+        containerClass: 'invoice-container',
+        headerClass: 'invoice-header',
+        logoClass: 'invoice-logo',
+        titleClass: 'invoice-title',
+        detailsClass: 'invoice-details',
+        bankDetailsClass:'bank-detail',
+        sellerInfoClass: 'seller-info',
+        clientInfoClass:'client-info',
```

Biome (1.9.4)

[error] This property value named clientInfoClass is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named clientInfoClass


(lint/suspicious/noDuplicateObjectKeys)